base/libgit2/types.jl

```diff
@@ -174,7 +174,50 @@ function RemoteCallbacks(credentials::Ptr{Void}, payload::Ref{Nullable{AbstractC
     RemoteCallbacks(credentials=credentials_cb(), payload=pointer_from_objref(payload))
 end
 
-if LibGit2.version() >= v"0.24.0"
+immutable ProxyOptions
```

Comment:
I don't think semver cares about internal names. This is non-breaking at the API level to keep this slightly closer to master.